### PR TITLE
13 component project image box

### PR DESCRIPTION
**Pull request by EvEliFr**

### CODE REVIEW

**Review by ESPR07 — CHANGES_REQUESTED** [review]

File: src/pages/ProjectDetailsPage.tsx [hunk 1]
```diff
@@ -34,14 +35,15 @@ const ProjectDetailsPage: React.FC = () => {
         <Meta title={"Your profile"}></Meta>
         <div className="projectDetailoutline">
           <div className="Title">{project?.title}</div>
+          <ProjectImageBox source={imageIcon} altText={"String"} />
```
[the remark this answers is not in the document]
Comment (ESPR07) [reply src/pages/ProjectDetailsPage.tsx:38]: I realize now that once the other image and profile thingy is removed it will be in the right spot >.<

**Review by ESPR07 — APPROVED** [review]

File: src/styles/ProjectImageBox.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.projectImageBox {
+    display: flex;
+    justify-content: center;
+}
+.projectImageBox img {
+    border-radius: 4px;;
```
Comment (ESPR07) [inline src/styles/ProjectImageBox.css:6]: This also has two ;; in it, cause the image to not have a border radius

Comment (EvEliFr) [reply src/styles/ProjectImageBox.css:6]: Sorry, I'm not very used to Git yet. Does this mean that you deleted the extra semicolon and I can just merge this, or should I fix it and push the fix again?

Comment (ESPR07) [reply src/styles/ProjectImageBox.css:6]: ah no, a review just means that I put comments on certain section of your changes, I don't make the changes myself. It's more of a "hey this looks incorrect, can you fix it and push again" kinda thing ^^

Comment (EvEliFr) [reply src/styles/ProjectImageBox.css:6]: I can't see that in my code... it won't let me commit since I haven't made any changes.

Comment (ESPR07) [reply src/styles/ProjectImageBox.css:6]: I actually noticed that too when I cloned so I dunno what happened >.<

Comment (EvEliFr) [reply src/styles/ProjectImageBox.css:6]: should I just merge it then?

Comment (ESPR07) [reply src/styles/ProjectImageBox.css:6]: Maybe you should remove the static image and profile container first, so that it becomes fully correct?

**Review by AKissMail — COMMENTED** [review]
Looks good on my system.

**Review by AKissMail — APPROVED** [review]

**Pull request merged**